uber/templates/jobs/staffers.html

```diff
@@ -70,7 +70,9 @@
 <div style="margin-left:10px">
     [D]: <i>Staffers marked with a "D" are department heads.</i> <br/>
     [M]: <i>Staffers marked with a "M" are assigned to multiple locations</i> <br/>
-    [T]: <i>Staffers marked with a "T" are trusted</i> <br/>
+    [T]: <i>Staffers marked with a "T" are trusted
+         {% if location %}in this department{% else %}in at least one department{% endif %}
+         </i> <br/>
     [L]: <i>Staffers marked with a "L" are working load-in and/or load-out</i>
```

Comment:
The way you have it now looks good to me; I don't think you need to implement either of the variants you suggested, since what's here now seems good to my eyes.